src/test/java/io/github/mfvanek/pg/support/PostgresSqlClusterWrapper.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2019-2022. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health
+ *
+ * This file is a part of "pg-index-health" - a Java library for
+ * analyzing and maintaining indexes health in PostgreSQL databases.
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.support;
+
+import io.github.mfvanek.pg.model.MemoryUnit;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+import javax.annotation.Nonnull;
+import javax.sql.DataSource;
+
+/**
+ * This wrapper provides postgres containers as part of HA cluster with repmgr configured.
+ * <p>
+ * If master goes down, repmgr will ensure any of the standby nodes takes the primary role.
+ *
+ * @author Alexey Antipin
+ * @since 0.6.2
+ */
+final class PostgresSqlClusterWrapper {
+
+    private static final String IMAGE_NAME = "docker.io/bitnami/postgresql-repmgr";
+    private static final String IMAGE_TAG = "14";
```

Comment:
Use here the same logic as in **PostgreSqlContainerWrapper**
```
    @Nonnull
    private static String preparePostgresVersion() {
        return Optional.ofNullable(System.getenv("TEST_PG_VERSION"))
                .orElse("14.5");
    }
```
[Bitnami doesn't provide image for 9.6](https://github.com/bitnami/containers/tree/main/bitnami/postgresql-repmgr#supported-tags-and-respective-dockerfile-links) so we need to remove it from CI pipeline (see .github/workflows/tests.yml)